appcenter-data/ios/AppCenterReactNativeData/AppCenterReactNativeData.m

```diff
@@ -48,20 +54,41 @@ @implementation AppCenterReactNativeData
 
 static NSString *const kMSjsonValueKey = @"jsonValue";
 
-static NSString *const kMSPartitionKey = @"partition";
-
-static NSString *const kMSIDKey = @"id";
-
 static NSString *const kMSETagKey = @"eTag";
 
 static NSString *const kMSLastUpdatedDateKey = @"lastUpdatedDate";
 
 static NSString *const kMSIsFromDeviceCacheKey = @"isFromDeviceCache";
 
+static NSString *const kMSIDKey = @"id";
+
+static NSString *const kMSPartitionKey = @"partition";
+
 static NSString *const kMSTimeToLiveKey = @"timeToLive";
 
+static NSString *const kMSErrorKey = @"error";
+
+static NSString *const kMSPaginatedDocumentsIDKey = @"paginatedDocumentsId";
+
+static NSString *const kMSItemsKey = @"items";
+
+static NSString *const kMSCurrentPageKey = @"currentPage";
+
+static NSString *const kMSMessageKey = @"message";
+
 RCT_EXPORT_MODULE();
 
+- (instancetype)init {
+    if((self = [super init])) {
```

Comment:
Our coding style guideline is to have a single space between `if`/`else`/`for`/etc. keyword and the open parenthesis. Please fix all the occurrences in this file.
```suggestion
    if ((self = [super init])) {
```

Reply:
Fixed